021-special-repository-for-st-clients-based-on-example-clients.md

```diff
@@ -0,0 +1,61 @@
+# Special repository for ST clients based on example clients
+
+This proposal suggests creating a new repository for `Strimzi` ST client based on 
+[Strimzi client-examples](https://github.com/strimzi/client-examples).
+
+## Current situation
+
+Currently, we are using two clients in our STs:
+ - `InternalKafkaClients` (based on `test-client` image) 
+ - `example clients` (from [Strimzi client-examples](https://github.com/strimzi/client-examples)).
+
+The plan is to remove the `InternalKafkaClients` and keep only `example clients`.
+
+## Motivation
+
+While testing `Strimzi` we need, in some cases, special configuration of clients, which is not implemented in the
+`client-examples`.
+How we discussed earlier, the `client-examples` repository should be really _exemplary_, and we should not add any extra
+_configuration_ or _extensions_ to it. For this kind of enhancements we should have repository, which will have
+`client-examples` as base, and we will be able to add special setting without disrupting the basic idea of example
+clients.
+
+## Proposal
```

Comment:
Maybe you could put there more details? Like repo name, etc. Maybe also suggest component owners?